README.rst

```diff
@@ -93,6 +93,10 @@ For example::
 
     towncrier create 123.feature
 
+If a news fragment is not tied to an issue, use `+` as the basename (a random hash will be added to the filename to keep it unique):
```

Comment:
Do we need to also talk about this in the README?

The risk is updating the implementation in the future, and forgeting to update the docs in all places.

I think that keeping README short is a good goal.

We have the docs for a full explanation of what towncrier can do.

Reply:
I agree that it's probably not the best place, but I think the reason I put it here is there *isn't* any other documentation section on `create`.